specification/batch/data-plane/Microsoft.Batch/stable/2021-06-01.14.0/BatchService.json

```diff
@@ -14197,14 +14197,14 @@
       "title": "Specifies the ephemeral disk placement for operating system disk for all compute nodes (VMs) in the pool.",
       "description": "This property can be used by user in the request to choose which location the operating system should be in. e.g., cache disk space for Ephemeral OS disk provisioning. For more information on Ephemeral OS disk size requirements, please refer to Ephemeral OS disk size requirements for Windows VMs at https://docs.microsoft.com/en-us/azure/virtual-machines/windows/ephemeral-os-disks#size-requirements and Linux VMs at https://docs.microsoft.com/en-us/azure/virtual-machines/linux/ephemeral-os-disks#size-requirements",
       "enum": [
-        "cachedisk"
+        "cacheDisk"
```

Comment:
Unless the REST API is actually going to return "cacheDisk" and not "cachedisk", this should be added as an exception to cspell.

Reply:
I couldn't find any instances of the all-lowers case in the service code. And actually, they're all 'CacheDisk', including the same enum for the control plane, so I think I'm going to change it to that.